recipes/xorg-xcb-proto/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set xorg_name = "xcb-proto" %}
+{% set xorg_category = "xcb" %}
+{% set name = "xorg-" ~ xorg_name %}
+{% set version = "1.12" %}
+{% set sha256 = "5922aba4c664ab7899a29d92ea91a87aa4c1fc7eb5ee550325c3216c480a4906" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - m2w64-pkg-config  # [win]
+    - pkg-config  # [not win]
+    - posix  # [win]
+    - python >=2.7,<3
```

Comment:
Similar question. Can we add skips for these versions of Python. Also, does it not build with Python 3?